conf/messages

```diff
@@ -31,15 +31,11 @@ common.yourIncomeTaxReturn.agent = Your client’s Income Tax Return
 common.caption = 6 April {0} to 5 April {1}
 startPage1 = Use this service to update and submit an Income Tax Return.
 startPage2 = This is a new service. At the moment you can only update information about:
-startPage.interestBullet.agent = interest paid to your client in the UK
-startPage.interestBullet.individual = interest paid to you in the UK
-startPage.dividendsBullet = dividends from UK companies, trusts and open-ended investment companies
-startPage.charityBullet.agent = your client’s donations to charity
-startPage.charityBullet.individual = your donations to charity
-startPage.employment.agent = You can view your client’s employment information but cannot use this service to update it until 6 April {0}.
-startPage.employment.individual = You can view your employment information but cannot use this service to update it until 6 April {0}.
-startPage3.agent.true = To update your client’s self-employment and property income, you must use your chosen commercial software.
-startPage3.agent.false = To update your self-employment and property income, you must use your chosen commercial software.
+startPage.interestBullet = UK interest
+startPage.dividendsBullet = dividends from UK-based companies, unit trusts and open-ended investment companies
+startPage.charityBullet = donations to charity
+startPage.paragraph.1 = You can view PAYE employment information, but you cannot update it until 6 April {0}.
```

Comment:
Should this be "your PAYE information"? I understand that it's not applicable to agents and individuals but nothing in the story to suggest otherwise currently